### ISSUE TITLE
[CS:GO StatTrak] Query failed

### ISSUE THREAD

**Issue by ciropavic** [TheDoctor0/CSGOMod#48]
[CS:GO StatTrak] Query failed. [1364] Field 'steamid' doesn't have a default value
 [CS:GO Accounts] Query failed. [1364] Field 'steamid' doesn't have a default value

**Comment by TheDoctor0** [comment]
Are you using version 2.1.0?

As documented under [CREATE TABLE](https://dev.mysql.com/doc/refman/8.0/en/create-table.html) syntax for MySQL:
> If neither NULL nor NOT NULL is specified, the column is treated as though NULL had been specified.

This means that the following column definitions are all identical:
```
columnname type NULL
columnname type DEFAULT NULL
```

First variant is already used in both StatTrak and Accounts table definitions:
```
CREATE TABLE IF NOT EXISTS `csgo_stattrak` (`name` VARCHAR(64) NULL, `steamid` VARCHAR(35) NULL, 

CREATE TABLE IF NOT EXISTS `csgo_accounts` (`name` VARCHAR(64) NULL, `steamid` VARCHAR(35) NULL, `pass` VARCHAR(32), PRIMARY KEY(name, steamid));
```

**Comment by TheDoctor0** [comment]
Fixed in version [2.1.1](https://github.com/TheDoctor0/CSGOMod/releases/tag/2.1.1).
If you suffer from this error use `csgo_reset_data` command or remove all tables manually as they need to be recreated.
